webui/react/src/pages/ModelVersionDetails.tsx

```diff
@@ -219,17 +222,17 @@ const ModelVersionDetails: React.FC = () => {
     }));
   }, [ modelVersion?.checkpoint ]);
 
-  if (isNaN(parseInt(modelId))) {
-    return <Message title={`Invalid Model ID ${modelId}`} />;
+  if (modelName === '') {
+    return <Message title={`Invalid Model Name ${modelName}`} />;
```

Comment:
Same thing about the error message here

Reply:
ok I am going to replace these both with
```javascript
  if (!modelName) {
    return <Message title={`Model name is empty`} />;
  }
```